bug: CI failed with "Unable to locate command >iverilog<"

The details is log [here](https://github.com/Readon/SpinalHDL/actions/runs/3627510211/jobs/6118569392).

I think the source of the error is:

```
2022-12-06T09:16:18.9454195Z parse.cc:299:10: fatal error: parse.hh: No such file or directory
2022-12-06T09:16:18.9454804Z   299 | #include "parse.hh"
2022-12-06T09:16:18.9455502Z       |          ^~~~~~~~~~
2022-12-06T09:16:18.9456369Z compilation terminated.
2022-12-06T09:16:18.9478366Z make[1]: *** [Makefile:108: parse.o] Error 1
```

This might be interesting: https://github.com/steveicarus/iverilog/issues/247#issuecomment-579806944

@kleinai The CI builds iverilog here: https://github.com/SpinalHDL/SpinalHDL/blob/dev/tools.sh#L29 sourced from https://github.com/SpinalHDL/SpinalHDL/blob/dev/.github/actions/action.yml#L49

> I think the source of the error is:
> 
> ```
> 2022-12-06T09:16:18.9454195Z parse.cc:299:10: fatal error: parse.hh: No such file or directory
> 2022-12-06T09:16:18.9454804Z   299 | #include "parse.hh"
> 2022-12-06T09:16:18.9455502Z       |          ^~~~~~~~~~
> 2022-12-06T09:16:18.9456369Z compilation terminated.
> 2022-12-06T09:16:18.9478366Z make[1]: *** [Makefile:108: parse.o] Error 1
> ```
> 
> This might be interesting: [steveicarus/iverilog#247 (comment)](https://github.com/steveicarus/iverilog/issues/247#issuecomment-579806944)
> 
> @kleinai The CI builds iverilog here: https://github.com/SpinalHDL/SpinalHDL/blob/dev/tools.sh#L29 sourced from https://github.com/SpinalHDL/SpinalHDL/blob/dev/.github/actions/action.yml#L49

Does it mean that the version of iverilog should be updated?

--- a/tools.sh
+++ b/tools.sh
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 install_verilator(){
-  sudo apt install -y git make autoconf g++ flex libfl-dev bison  # First time prerequisites
+  sudo apt install -y git make autoconf build-essential flex libfl-dev bison  # First time prerequisites
   git clone http://git.veripool.org/git/verilator   # Only first time
   unset VERILATOR_ROOT  # For bash
   cd verilator
@@ -15,7 +15,7 @@ install_verilator(){
 }
 
 install_ghdl(){
-  sudo apt install -y gnat-9  libgnat-9 zlib1g-dev libboost-dev
+  sudo apt install -y gnat-9 libgnat-9 zlib1g-dev libboost-dev
   git clone https://github.com/ghdl/ghdl ghdl-build && cd ghdl-build
   git reset --hard "v1.0.0"
   mkdir build
@@ -27,14 +27,7 @@ install_ghdl(){
 }
 
 install_iverilog(){
-  sudo apt install -y gperf readline-common bison flex libfl-dev
-  curl -fsSL https://github.com/steveicarus/iverilog/archive/v10_3.tar.gz | tar -xvz
-  cd iverilog-10_3
-  autoconf
-  ./configure  --prefix ~/tools
-  make -j$(nproc)
-  make install
-  cd ..
+  sudo apt install -y iverilog
 }
 
 install_cocotb(){

